lib/workers/repository/onboarding/pr/config-description.ts

```diff
@@ -47,9 +42,9 @@ export function getConfigDesc(
   const enabledManagers = packageFiles ? Object.keys(packageFiles) : [];
   if (
     !(
-      enabledManagers.includes(MANAGER_DOCKERFILE) ||
-      enabledManagers.includes(MANAGER_CIRCLE_CI) ||
-      enabledManagers.includes(MANAGER_DOCKER_COMPOSE)
+      enabledManagers.includes('dockerfile') ||
+      enabledManagers.includes('circleci') ||
+      enabledManagers.includes('docker-compose')
```

Comment:
This rule can probably go away actually. We never ended up using this "odd is unstable" concept that the preset refers to.

Reply:
I don't actually understand what's going on here. Are we deleting whole `if`-branch here?